Reply/app/src/main/java/com/materialstudies/reply/util/BindingAdapter.kt

```diff
@@ -74,6 +76,14 @@ fun TextView.bindDrawables(
     )
 }
 
+@BindingAdapter(
+    "backgroundSurfaceElevation"
```

Comment:
`"backgroundElevationOverlay"` and `View.bindBackgroundElevationOverly(elevation: Float)`?

Reply:
Forgot about this. Sg